### ISSUE TITLE
DOF R3 Beta 1 firing events on PacLed64 randomly

### ISSUE THREAD

**Issue by xantari** [DirectOutput/DirectOutput#5]
See here:

http://vpuniverse.com/forums/forums/topic/2765-how-to-limit-knocker-usage-in-certain-tables/

Basically what appears to be happening is that the PacLed64 driven virtual output kit from Zebsboards.com is firing events a lot more in DOF R3 than in DOF R2.

I have reverted back to DOF R2 for now, but it appears there might be an issue with the new PacLed64 code in DOF R3.

Not only do knockers fire off wildly, I also noticed brief LED light up during table initliazation that activates an LED light upon bootup of STTNG, these brief momentary lightups do not occur on DOF R2. I can provide a video that effect if necessary.

**Comment by xantari** [comment]
Hi Swisslizard, I updated the forum post in regards to this issue. The new build didn't fix the issue. I also added a bunch more information to the post in hopes of narrowing down what is going on with this particular table/ROM combination that didn't happen in DOF R2. 

If you don't have a PacLed64 i'm willing to buy you one if needed. Just ping me and let me know.

**Comment by xantari** [comment]
One other thing, look at the new video I posted. As I think it's related to the knocker issue. Very simple test case, pressing the right flipper button on this game randomly lights LED lights very briefly for a split second if you press them really fast. DOF R3 seems to be picking up a lot more events that perhaps R2 ignored in the past or some sort of strobing effect is happening on this ROM that affects the outputs. Just guesses, tried to debug but logging is to minimal to pinpoint the problem.

**Comment by lizardking** [comment]
After various tests with xantari we condluded that this is not a DOF issue but rather a problem wihich is cause by usb hardware, usb driver, output controller firmware and timeing of data sent to the output controller (lieke a combination of them). 
As far is I can tell there is not DOF can do about this.
Nevertheless I have added some addition config properties for the Pacled64 which allow to change the timing by adding a slight delay. This ,oght helps in some cases.
